src/objectExplorer/objectExplorerService.ts

```diff
@@ -27,6 +27,15 @@ import { ConnectionProfile } from '../models/connectionProfile';
 import providerSettings from '../azure/providerSettings';
 import { IConnectionInfo } from 'vscode-mssql';
 
+function getParentNode(node: TreeNodeType): TreeNodeInfo {
+    node = node.parentNode;
+    if (!(node instanceof TreeNodeInfo)) {
+        vscode.window.showErrorMessage(LocalizedConstants.nodeErrorMessage);
+        throw new Error(`Something went wrong when trying to find the correct node.`);
```

Comment:
You've verified that this isn't displayed to the user?

I'd also make it a little clearer what the actual problem is - something like `Parent node was not TreeNodeInfo`. (and if there's any general name or something you can print as well for the current node that'd definitely be useful)

Reply:
yes I've verified this - it shows up in the debug console